docs/source/development/ISISSANSReductionBackend.rst

```diff
@@ -0,0 +1,1212 @@
+- ``SANSNormalizeToMonitor``
+- ``SANSSave``
+- ``SANSScale``
+- ``SANSSliceEvent``
+
+Note that the vast majority of the these algorithms takes a *SANSState* object as
+an input.
+
+The individual algorithms are superficially discussed below.
+
+There are two further algorithms which coordinate these algorithms, they are ``SANSReductionCore``
+and ``SANSSingleReduction`` which are discussed further down.
+
+
+``SANSCalculateTransmission``
+------------------------------
+
+The ``SANSCalculateTransmission`` algorithm is one of the more complex algorithms
+in the reduction chain with many sub-steps and a wide variety of parameters which
+can be set by the users.
+
+The algorithm performs the following steps:
+
+1. Select the incident monitor. If this is not explicitly set then the default value is taken.
+2. Select the transmission detector ids. The detector ids are chosen to via the following preference:
+
+   a. If available, get detector ids from region-of-interest selection on detector
+   b. Else if available get detector ids from transmission monitor setting
+   c. Else get default transmission monitor
+
+3. Get the corrected transmission workspace. The sub-steps are:
+
+   a. Load the transmission workspace
+   b. Extract the transmission detector ids with :ref:`ExtractSpectra <algm-ExtractSpectra>`
+   c. Perform prompt peak correction
+   d. Perform flat background correction to monitors (if applicable) using :ref:`CalculateFlatBackground <algm-CalculateFlatBackground>`
+   e. Perform flat background correction to other detectors (if applicable) using :ref:`CalculateFlatBackground <algm-CalculateFlatBackground>`
+   f. Convert to wavelength and rebin using :ref:`SANSConvertToWavelengthAndRebin <algm-SANSConvertToWavelengthAndRebin>`
+
+4. Get the corrected direct workspace. The sub-steps are:
+
+   a. Load the transmission workspace
+   b. Extract the transmission detector ids with :ref:`ExtractSpectra <algm-ExtractSpectra>`
+   c. Perform prompt peak correction (if applicable) using :ref:`RemoveBins <algm-RemoveBins>`
+   d. Perform flat background correction to monitors (if applicable) using :ref:`CalculateFlatBackground <algm-CalculateFlatBackground>`
+   e. Perform flat background correction to other detectors (if applicable) using :ref:`CalculateFlatBackground <algm-CalculateFlatBackground>`
+   f. Convert to wavelength and rebin using :ref:`SANSConvertToWavelengthAndRebin <algm-SANSConvertToWavelengthAndRebin>`
+
+5. Perform fitting for the transmission calculation. The sub-steps are:
+
+   a. Use incident monitor, wavelength settings, transmission detector ids
+      and fit settings to initialize :ref:`CalculateTransmission <algm-CalculateTransmission>`
+   b. Execute :ref:`CalculateTransmission <algm-CalculateTransmission>`
+   c. Get the fitted and unfitted workspace
+
+6. Set the fitted and unfitted workspaces on the output of the algorithm.
+
+
+``SANSConvertToQ``
+------------------
+
+The ``SANSConvertToQ`` algorithm is the
```

Comment:
N \cross M  maybe \times works in this case (cross product)?

I get:

System Message: WARNING/2 (N \cross M)

latex exited with error [stdout] This is pdfTeX, ...